plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduPageSink.java

```diff
@@ -144,6 +141,11 @@ private void appendColumn(PartialRow row, Page page, int position, int channel,
         else if (TIMESTAMP_MILLIS.equals(type)) {
             row.addLong(destChannel, truncateEpochMicrosToMillis(type.getLong(block, position)));
         }
+        else if (DATE.equals(type)) {
+            SqlDate date = (SqlDate) type.getObjectValue(connectorSession, block, position);
+            LocalDate dt = LocalDate.ofEpochDay(date.getDays());
+            row.addDate(destChannel, Date.valueOf(dt));
```

Comment:
java.sql.Date doesn't handle DST gaps correctly. Does kudu-client accept any other type (LocalDate or String)?

Reply:
I see Cassandra connector uses `io.trino.spi.type.DateType` to handle date type in method `io.trino.plugin.cassandra.CassandraPageSink#appendColumn`, maybe we could take this as an example.

Reply:
Changed it to use the getInt API